includes/classes/Indexable/Post/Post.php

```diff
@@ -799,19 +806,20 @@ function( $tax_query ) {
 		/**
 		 * Try to find other taxonomies set in the root of WP_Query
 		 *
-		 * @since  3.4
+		 * @since 3.4
+		 * @since 3.4.2 Test taxonomies with their query_var value.
 		 */
-		$taxonomies = get_taxonomies();
+		$taxonomies = get_taxonomies( array(), 'objects' );
 
-		foreach ( $taxonomies as $tax_slug ) {
+		foreach ( $taxonomies as $tax_slug => $tax ) {
 			if ( 'ep_custom_result' === $tax_slug || $this->is_protected_parameter( $tax_slug ) ) {
 				continue;
 			}
```

Comment:
They have removed this block upstream, I left it because we had added `is_protected_parameter`

They changed `$tax_slug` for `$tax->query_var` upstream, we might need to investigate if it would be reasonable to do the same in this block.

Reply:
Seems OK to keep that block based on my limited knowledge.

`$tax_slug` vs `$tax->query_var`, who is meant to investigate that? Do you feel we should check that before we merge that? I am happy to test it locally and if it works + passes the tests approve. 

I guess reviewing the whole change would be tough. Only some merge conflict resolutions between our fork and upstream are probably relevant for us right? Did you have to do some?